icu4c/source/i18n/unicode/ucol.h

```diff
@@ -1519,6 +1519,130 @@ ucol_openBinary(const uint8_t *bin, int32_t length,
                 const UCollator *base, 
                 UErrorCode *status);
 
+#if U_SHOW_CPLUSPLUS_API
```

Comment:
In this PR after rebasing, or in a follow-up PR, we should also add ` || U_SHOW_CPLUSPLUS_HEADER_API`

Reply:
Done.